src/js/utils/utils.js

```diff
@@ -44,3 +44,17 @@ export function getLastModifiedDate(intl, modfiedDate) {
     var value = { date: formattedModefiedDate };
     return formatMessage({ id: 'utils.utils.lastmodifieddate' }, value);
 }
+
+export function humanFileSize(bytes) {
```

Comment:
If this is only used by Analytics I'd rather move it into the container.  I don't like util classes because it isn't clear where they're being used.  Up to you though.